runtime/power.S

```diff
@@ -1193,6 +1193,7 @@ caml_system.frametable:
         .short  -1              /* negative frame size => use callback link */
         .short  0               /* no roots here */
         .align 3
+        .size   caml_system.frametable, .-caml_system.frametable
```

Comment:
For consistency with other platforms, you should probably add a `ENDOBJECT` macro and use it here.

Reply:
I expect my original thinking (in 2022) was along the lines that there was no alternate version so it didn't need a macro... how does 7be1ad034dbe74ae5c26e74f05ceb38d49e87f7b (putting in both `OBJECT` and `ENDOBJECT` to both) look?

Reply:
Almost good! The `.section` directives should be left outside of these macros, however.

Reply:
Done (finally - sorry for the delay)